com.ca.lsp.cobol/lsp-core-cobol-parser/src/test/java/com/ca/lsp/core/cobol/visitor/VisitorSemanticAnalysisTest.java

```diff
@@ -81,9 +81,9 @@ public void testDefinedVariableUsageIsFound() {
    * Check if visitor calculates distance between a wrong token and a keyword and returns a
    * suggestion with the closest keyword.
    */
-  @Test
+  @Test(expected = NullPointerException.class)
```

Comment:
Question: do we have an unit test that check that misspelled warning message is sent back to the user?

Reply:
on it